app/src/test/java/dev/andrylat/raqimbek/bankingutils/BankingUtilsAppTest.java

```diff
@@ -0,0 +1,52 @@
+package dev.andrylat.raqimbek.bankingutils;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import org.junit.jupiter.api.Test;
+
+public class BankingUtilsAppTest {
```

Comment:
Unit tests must cover all classes except POJO.
It means that you need to cover validator in the separate class, payment system determiner, etc.